lib/services/workload_identity.go

```diff
@@ -75,6 +76,11 @@ func UnmarshalWorkloadIdentity(
 	return UnmarshalProtoResource[*workloadidentityv1pb.WorkloadIdentity](data, opts...)
 }
 
+const (
+	maxMaxJWTSVIDTTL  = time.Hour * 24
+	maxMaxX509SVIDTTL = time.Hour * 24 * 14
```

Comment:
Is this a typo or intentionally `maxMax`, as in the maximum max value that can be configured?

Reply:
Yeah - it's the maximum max value that can be configured.